test_submissions/civic/batch.json

```diff
@@ -0,0 +1,224 @@
+      },
+      "clinicalImpactClassification": {
+        "clinicalImpactClassificationDescription": "Tier II - Potential",
+        "assertionTypeForClinicalImpact": "therapeutic: sensitivity/response",
+        "drugForTherapeuticAssertion": "vemurafenib",
+        "comment": "Phase 3 randomized clinical trial comparing vemurafenib with dacarbazine in 675 patients with previously untreated, metastatic melanoma with the BRAF V600E mutation. At 6 months, overall survival was 84% (95% confidence interval [CI], 78 to 89) in the vemurafenib group and 64% (95% CI, 56 to 73) in the dacarbazine group. A relative reduction of 63% in the risk of death and of 74% in the risk of either death or disease progression was observed with vemurafenib as compared with dacarbazine (P<0.001 for both comparisons).",
+        "citation": [
+          {
+            "db": "PubMed",
+            "id": "21639808"
+          }
+        ]
+      }
+    },
+    {
+      "recordStatus": "novel",
+      "localID": "civic.mpid:12",
+      "localKey": "civic.eid:1398",
+      "observedIn": [
+        {
+          "alleleOrigin": "somatic",
+          "affectedStatus": "yes",
+          "collectionMethod": "curation"
+        }
+      ],
+      "conditionSet": {
+        "condition": [
+          {
+            "db": "MONDO",
+            "id": "0005105"
+          }
+        ]
+      },
+      "variantSet": {
+        "variant": [
+          {
+            "hgvs": "ENST00000288602.6:c.1799T>A",
+            "gene": [
+              {
+                "id": 673
+              }
+            ]
+          }
+        ]
+      },
+      "clinicalImpactClassification": {
+        "clinicalImpactClassificationDescription": "Tier I - Strong",
+        "assertionTypeForClinicalImpact": "therapeutic: sensitivity/response",
+        "drugForTherapeuticAssertion": "vemurafenib",
+        "comment": "The BRIM-3 Phase III trial NCT01006980 assessed BRAF inhibitor vemurafenib versus dacarbazine in 598 patients with treatment naive metastatic melanoma and confirmed V600E mutation. Significant differences were seen in overall survival (13.3 months with vemurafenib vs. 10.0 months with dacarbazine) and median progression free survival (6.9 months  with vemurafenib vs. 1.6 months with dacarbazine)",
+        "citation": [
+          {
+            "db": "PubMed",
+            "id": "24508103"
+          }
+        ]
+      }
+    },
+    {
+      "recordStatus": "novel",
+      "localID": "civic.mpid:12",
+      "localKey": "civic.eid:3017",
+      "observedIn": [
+        {
+          "alleleOrigin": "somatic",
+          "affectedStatus": "yes",
+          "collectionMethod": "curation"
+        }
+      ],
+      "conditionSet": {
+        "condition": [
+          {
+            "db": "MONDO",
+            "id": "0005233"
+          }
+        ]
+      },
+      "variantSet": {
+        "variant": [
+          {
+            "hgvs": "ENST00000288602.6:c.1799T>A",
+            "gene": [
```

Comment:
@korikuzma Is the gene blank or am I missing it?

Reply:
It's below that line

Reply:
is it just the ID? I am not seeing a gene symbol.

Reply:
Yup, only one is required as seen [here](https://github.com/ncbi/clinvar/blob/2184a4288bf53de66767ef83fdd1b0f7a4145b48/submission_api_schema/submission_apitest_schema.json#L779-L790).

Reply:
Gotcha just making sure.